dev-lang/nim/nim-1.6.6-r2.ebuild

```diff
@@ -15,11 +15,10 @@ SRC_URI="
 LICENSE="MIT"
 SLOT="0"
 KEYWORDS="~amd64 ~arm ~x86"
-IUSE="debug experimental +readline"
+IUSE="debug experimental"
 RESTRICT="test"  # need to sort out depends and numerous failures
 
-RDEPEND="readline? ( sys-libs/readline:0= )"
-DEPEND="${RDEPEND}"
+BDEPEND="<sys-process/parallel-20220622"  # bug 856079
```

Comment:
I don't get it.
This one: https://bugs.gentoo.org/856079 ?

Reply:
It's a bug which needs fixing but this is really not the right fix. You could argue this for every single reverse dep of parallel.

Reply:
Yes

Reply:
Fixed properly now. In future, just ping. Please no hacks like this.

Reply:
Removed the hack

Reply:
thanks!